### ISSUE TITLE
feat(Slider): add isDisabled property to prohibit any interaction with the component

### ISSUE THREAD

**Issue by KKoukiou** [patternfly/patternfly-react#5459]

**Comment by tlabaj** [comment]
cc @mcarrano

**Comment by mcarrano** [comment]
Yeah, I guess that it's true that we didn't create a disabled state for the slider. @mcoker do you recall if we discussed this? In any case I think we will need to define the additional styles in Core before we can make the React update.

**Comment by mcoker** [comment]
@mcarrano nope, not that I recall.

For now, if a user has a value input wither beside or on the slider thumb, `isDisabled` could disable that, then disable any interaction with the slider, and the disabled text input is the visual indication that the slider is disabled.

Though for a slider without a text input, all `isDisabled` would be able to do is disable the interaction. There would be no other visual indication that the slider is disabled.

If we want to add visual changes to the slider itself when `isDisabled` is applied, we would need to add that to core.

**Comment by mcarrano** [comment]
What is your priority for this issue @KKoukiou ? I think my preference would be to move this to core and work with @mcoker over the next sprint to implement a disabled visual style to the component before implementing the React prop. But if you need this immediately then the suggestion above provides some workaround.

**Comment by KKoukiou** [comment]
Slider is the last component we use from PF3 in some pages in cockpit. It would be nice to stop importing PF3, but really it's not so urgent. I can also help with the react implementation in this repo, once core is ready.

**Comment by mcarrano** [comment]
Thanks @KKoukiou . I am going to transfer this over to core then to add the disabled state in the upcoming sprint with the expectation that we can hand it back to you to make the React update following that. Given that this is small, I think it's feasible to get everything done in one sprint.

@mcoker, rather than open a design issue for this, perhaps we can just get together and work out what the disabled styling will be? Does that work for you?

**Comment by mcoker** [comment]
@mcarrano works for me 👍
